### PR TITLE
POA Not Found updates

### PR DESCRIPTION
**Pull request by Jeremy1026**
### Description
Updates to Power of Attorney section of Case Details when :not_found.

### Acceptance Criteria
- [ ] Code compiles correctly

### CODE REVIEW

File: client/app/queue/PowerOfAttorneyDetail.jsx [hunk 1]
```diff
@@ -121,6 +117,8 @@ export const PowerOfAttorneyDetailUnconnected = ({ powerOfAttorney, appealId, po
   }
 
   const showPoaDetails = poa.representative_type && poa.representative_name;
+  const bototmMessage = (showPoaDetails || poaAlert.powerOfAttorney) ? COPY.CASE_DETAILS_POA_EXPLAINER :
```
Comment (ajspotts) [inline client/app/queue/PowerOfAttorneyDetail.jsx:120]: should this be `bottomMessage` ?

Comment (ajspotts) [inline client/app/queue/PowerOfAttorneyDetail.jsx:120]:
```suggestion
  const bottomMessage = (showPoaDetails || poaAlert.powerOfAttorney) ? COPY.CASE_DETAILS_POA_EXPLAINER :
```

File: client/app/queue/PowerOfAttorneyDetail.jsx [hunk 2]
```diff
@@ -134,7 +132,7 @@ export const PowerOfAttorneyDetailUnconnected = ({ powerOfAttorney, appealId, po
             <BareList ListElementComponent="ul" items={details.map(getDetailField)} />
           </ul>
         )}
-        <p><em>{ COPY.CASE_DETAILS_POA_EXPLAINER }</em></p>
+        <p><em>{ bototmMessage }</em></p>
```
Comment (ajspotts) [inline client/app/queue/PowerOfAttorneyDetail.jsx:135]:
```suggestion
        <p><em>{ bottomMessage }</em></p>
```

File: client/app/queue/components/PoaRefresh.jsx [hunk 3]
```diff
@@ -35,9 +35,12 @@ export const PoaRefresh = ({ powerOfAttorney, appealId }) => {
   return <React.Fragment>
     {viewPoaRefresh &&
     <div {...textStyling}>
-      <em>Power of Attorney (POA) data comes from VBMS. To refresh POA, please click the "Refresh POA" button.</em>
+      <em>To retrieve the latest POA information, please click the "Refresh POA" button.</em>
```
Comment (ajspotts) [inline client/app/queue/components/PoaRefresh.jsx:38]: Should we move this out to `COPY.json` similar to CASE_DETAILS_POA_EXPLAINER ?

File: spec/feature/queue/case_details_spec.rb [hunk 4]
```diff
@@ -568,6 +569,7 @@
         allow_any_instance_of(BgsPowerOfAttorney).to receive(:bgs_record).and_return(:not_found)
         click_on "Refresh POA"
         expect(page).to have_content("Successfully refreshed. No power of attorney information was found at this time.")
+        expect(page).to have_no_content("POA last refreshed on")
```
Comment (ajspotts) [inline spec/feature/queue/case_details_spec.rb:572]: If we are expecting the page to `have_no_content` what is the `"POA last refreshed on"` text we are passing in here?

**Comment by Jeremy1026** [comment]
Combined with #16231

**Pull request closed without merging**